apps/console/src/features/organizations/components/edit-organization-role/edit-organization-groups.tsx

```diff
@@ -0,0 +1,714 @@
+                            + "emptyPlaceholders.default") }
+                    >
+                        {
+                            tempGroupList?.map((group, index)=> {
+                                return (
+                                    <TransferListItem
+                                        handleItemChange={
+                                            () => handleAssignedItemCheckboxChange(group)
+                                        }
+                                        key={ index }
+                                        listItem={ resolveListItem(group?.displayName) }
+                                        listItemId={ group?.id }
+                                        listItemIndex={ index }
+                                        listItemTypeLabel={ resolveListItemLabel(group?.displayName) }
+                                        isItemChecked={ checkedAssignedListItems.includes(group) }
+                                        showSecondaryActions={ false }
+                                        data-testid="role-mgt-update-groups-modal-selected-groups"
+                                    />
+                                );
+                            })
+                        }
+                    </TransferList>
+                </TransferComponent>
+            </Modal.Content>
+            <Modal.Actions>
+                <Grid>
+                    <Grid.Row columns={ 2 }>
+                        <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
+                            <LinkButton
+                                data-testid="role-mgt-update-groups-modal-cancel-button"
+                                floated="left"
+                                onClick={ handleCloseAddNewGroupModal }
+                            >
+                                { t("common:cancel") }
+                            </LinkButton>
+                        </Grid.Column>
+                        <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 8 }>
+                            <PrimaryButton
+                                data-testid="role-mgt-update-groups-modal-save-button"
+                                floated="right"
+                                loading={ isSubmitting }
+                                disabled={ isSubmitting }
+                                onClick={ () => updateRoleGroup(role, tempGroupList) }
+                            >
+                                { t("common:save") }
+                            </PrimaryButton>
+                        </Grid.Column>
+                    </Grid.Row>
+                </Grid>
+            </Modal.Actions>
+        </Modal>
+    );
+
+    const handleAssignedGroupListSearch = (e, { value }) => {
+        let isMatch = false;
+        const filteredGroupList = [];
+
+        if (!isEmpty(value)) {
+            const re = new RegExp(escapeRegExp(value), "i");
```

Comment:
Shall we have a meaningful variable name?

Reply:
Fixed in `bd4c9bc5`